torch/distributed/_composable/fsdp/_fsdp_param.py

```diff
@@ -234,6 +235,7 @@ def _init_sharded_param(self, param: nn.Parameter, device: torch.device):
             self._global_stride = param.stride()
             param_data = param
         self._orig_size = param_data.size()
+        self._contiguous_orig_stride = make_contiguous_strides_for(self._orig_size)
```

Comment:
nit: curious if this is from rebasing since I removed this attribute at some point or if this is intentional to re-add

Reply:
sorry forgot to add review comment for this: this is intentional to avoid running into SymInt issues (which happens if `make_contiguous_strides_for()` is traced by Dynamo)